tiproxy/tiproxy-overview.md

```diff
@@ -0,0 +1,184 @@
+
+    A configuration example is as follows:
+
+    ```yaml
+    server_configs:
+      tidb:
+        security.session-token-signing-cert: "/var/sess/cert.pem"
+        security.session-token-signing-key: "/var/sess/key.pem"
+        graceful-wait-before-shutdown: 15
+    ```
+
+3. Configure the TiProxy instances.
+
+    To ensure the high availability of TiProxy, it is recommended to deploy at least two TiProxy instances. You can use hardware load balancers or configure virtual IP to distribute traffic to each TiProxy instance.
+
+    When selecting the model and number of TiProxy instances, consider the following factors:
+
+    - For the workload type and maximum QPS, see [TiProxy Performance Test Report](/tiproxy/tiproxy-performance-test.md).
+    - Because the number of TiProxy instances is less than that of TiDB servers, the network bandwidth of TiProxy is more likely to become a bottleneck than that of TiDB servers. Therefore, you also need to consider the network bandwidth. For example, in AWS, the baseline network bandwidth of the same series of EC2 is not proportional to the number of CPU cores. For details, see [Network performance](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/compute-optimized-instances.html#compute-network-performance). In such cases, when the network bandwidth becomes a bottleneck, splitting the TiProxy instance into more and smaller instances can improve QPS.
+
+    It is recommended to enable TLS connections within the cluster to avoid data leakage between TiProxy and TiDB server.
+
+    To configure TiProxy configuration items, see [TiProxy configuration](/tiproxy/tiproxy-configuration.md).
+
+    A configuration example is as follows:
+
+    ```yaml
+    global:
+      enable_tls: true
+    server_configs:
+      tiproxy:
+        proxy.addr: "0.0.0.0:6000"
+        api.addr: "0.0.0.0:3080"
+    ```
+
+4. Start the cluster.
+
+    To start the cluster using TiUP, see [TiUP documentation](/tiup/tiup-documentation-guide.md).
+
+5. Connect to TiProxy.
+
+    After the cluster is deployed, the cluster exposes the ports of TiDB server and TiProxy at the same time. The client should connect to the port of TiProxy instead of the port of TiDB server.
+
+### Modify TiProxy configuration
+
+To ensure that TiProxy keeps the client connection, TiProxy should not be restarted at will. Therefore, most of the TiProxy configuration items can be modified online. For the list of configuration items that support online change, see [TiProxy configuration](/tiproxy/tiproxy-configuration.md).
+
+When using TiUP to change the TiProxy configuration, if the configuration item to be changed supports online change, you should use the [`--skip-restart`](/tiup/tiup-component-cluster-reload.md#--skip-restart) option to avoid restarting TiProxy.
```

Comment:
```suggestion
When using TiUP to change the TiProxy configuration, if the configuration item to be changed supports online change, you can use the [`--skip-restart`](/tiup/tiup-component-cluster-reload.md#--skip-restart) option to avoid restarting TiProxy.
```